### PR TITLE
Give more structure to the files guide

### PR DESCRIPTION
**Pull request by KnorpelSenf**
This PQ breaks up the wall of text that bothers [some people](https://t.me/grammyjs/225729) by introducing intermediate headers.

### CODE REVIEW

File: site/docs/guide/files.md [hunk 1]
```diff
@@ -6,41 +6,66 @@ This involves handling the files that are attached to the messages.
...
+
 You can reuse the same `file_id` as often as you want, so you could send the same file to five different chats, using the same `file_id`.
 However, you must make sure to use the correct method---for example, you cannot use a `file_id` that identifies a photo when calling [`sendVideo`](https://core.telegram.org/bots/api#sendvideo).
 
+Files can be sent like [this](#sending-files).
+
+### Identifiers May Surprise You
+
+> File identifiers **only work for your bot**.
+> If another bot uses your file identifiers, it may randomly work and randomly crash and randomly kill innocent kittens.
```
Comment (roj1512) [inline site/docs/guide/files.md:49]: > it may randomly work

Didn’t know this! But yes, it’s technically possible.

Comment (KnorpelSenf) [reply site/docs/guide/files.md:49]: Indeed, and it happens that people run into this and then they are like “but it worked yesterday and now they're all dead” :( poor kittens, but we all know who's at fault here.

**Review by roj1512 — APPROVED** [review]

File: site/docs/uk/guide/files.md [hunk 2]
```diff
@@ -10,23 +10,45 @@
...
+
+> Боти отримують лише ідентифікатори файлів.
+> Якщо вони хочуть отримати вміст файлу, вони повинні запитати його явно.
 
 Кожного разу, коли ваш бот **отримує** повідомлення з файлом, він отримує не повні дані файлу, а лише його `file_id`.
 Якщо вашому боту потрібно завантажити файл, він може зробити це, викликавши метод `getFile` ([довідка Telegram Bot API](https://core.telegram.org/bots/api#getfile)).
 Цей метод дозволяє завантажити файл, створивши спеціальну тимчасову URL-адресу.
 Зауважте, що ця URL-адреса гарантовано дійсна лише протягом 60-ти хвилин, після чого вона може перестати працювати. У цьому випадку ви можете просто знову викликати `getFile`.
 
+Файли можна отримати ось [так](#отримання-фаилів).
```
Comment (LWJerri) [inline site/docs/uk/guide/files.md:25]: https://github.com/grammyjs/website/pull/1021/files#diff-556f3fdaec0d3e4c57f66f39f9b2c792ab9594f9848796f6cd0a60a6af371c57R44

File: site/docs/uk/guide/files.md [hunk 3]
```diff
@@ -10,23 +10,45 @@
...
+> Вони дуже ефективні.
 
 Коли бот надсилає повідомлення, він може **вказати `file_id`, який бачив раніше**.
 Це дозволить йому надіслати ідентифікований файл без необхідності завантажувати дані для нього.
-Щоб дізнатися, як завантажити власні файли, [прокрутіть униз](#надсилання-фаилів).
+
 Ви можете повторно використовувати той самий `file_id` скільки завгодно, наприклад, ви можете надіслати той самий файл до пʼяти різних чатів, використовуючи той самий `file_id`.
 Однак ви повинні переконатися, що використовуєте правильний метод: наприклад, ви не можете використовувати `file_id`, який ідентифікує фотографію, для виклику [`sendVideo`](https://core.telegram.org/bots/api#sendvideo).
 
+Файли можна надіслати ось [так](#надсилання-фаилів).
```
Comment (LWJerri) [inline site/docs/uk/guide/files.md:44]: https://github.com/grammyjs/website/pull/1021/files#diff-556f3fdaec0d3e4c57f66f39f9b2c792ab9594f9848796f6cd0a60a6af371c57R25

**Review by LWJerri — APPROVED** [review]
:shipit:

**Review by deptyped — APPROVED** [review]

**Review by dzikrl — APPROVED** [review]

**Review by quadratz — APPROVED** [review]

**Review by dzikrl — APPROVED** [review]
Well written.

**Pull request merged**